website/content/docs/connect/config-entries/service-resolver.mdx

```diff
@@ -512,20 +576,82 @@ spec:
+            'The namespace to resolve the requested service from to form the failover group of instances. If empty, the current namespace is used.',
         },
         {
           name: 'Datacenters',
           type: 'array<string>',
           description: 'A fixed list of datacenters to try during failover.',
         },
+        {
+          name: 'Targets',
+          type: 'array<ServiceResolverFailoverTarget>',
+          description: `A fixed list of failover targets to try during
+                          failover. It allows operators to express complicated failover
+                          scenarios such as between cluster peers, WAN federated datacenters, and local admin partitions.`,
+          children: [
+            {
+              name: 'Service',
+              type: 'string: ""',
+              description:
+                'The service name to use for the failover target. If empty, the current service name is used.',
+            },
+            {
+              name: 'ServiceSubset',
+              type: 'string: ""',
+              description:
+                'The named subset to use for the failover target. If empty, the default subset for the requested service name is used.',
+            },
+            {
+              name: 'Namespace',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                `The namespace to use for the failover target. If empty, the \`default\` namespace is used.`,
+            },
+            {
+              name: 'Partition',
+              enterprise: true,
+              type: 'string: ""',
+              description:
+                `The admin partition within the same datacenter to use for the failover target.
+                If empty, the \`default\` partition is used.
+                To use an admin partition in a different datacenter for the failover target,
+                use the \`Peer\` field instead.`,
+            },
+            {
+              name: 'Datacenter',
+              type: 'string: ""',
+              description:
+                `The WAN federated datacenter to use for the failover target.
+                If empty, the current datacenter is used.
+                To use a datacenter for the failover target that is connected
+                with a cluster peering relationship rather than WAN federation ,
+                use the \`Peer\` field instead.`,
+            },
+            {
+              name: 'Peer',
+              type: 'string: ""',
+              description:
+              `Specifies the destination cluster peer to resolve the target service name from.
+                  Ensure that [intentions are defined](/docs/connect/cluster-peering/create-manage-peering#authorize-services-for-peers)
+                  on the peered cluster to allow the source service to call the upstream service.
```

Comment:
@erichaberkorn : does this seem like a helpful clarification (that the upstream we're referring to is the service in this failover target).

```suggestion
                  on the peered cluster to allow the source service to access this failover target service as an upstream.
```